browser/ui/views/tabs/brave_tab_style_views.inc.cc

```diff
@@ -10,6 +10,21 @@ namespace {
 
 using tabs::features::HorizontalTabsUpdateEnabled;
 
+// Returns a value indicating if the browser frame view is "condensed", i.e.
+// that its frame border is somehow collapsed, as in fullscreen or when
+// maximized, or in Linux when caption buttons are not displayed.  For tabs,
```

Comment:
```suggestion
// maximized, or in Linux when caption buttons are not displayed. For tabs,
```

Reply:
Just double checking, is the difference between this function and ShouldExtendHitTest() that frame can be condensed on Linux when caption buttons are not displayed?

Reply:
Yes, this can be the case when using a "tiling" window manager on Linux. It's not a problem for our old tabs, because they extended to the top of the tabstrip view.